docs/products/flink/overview.rst

```diff
@@ -0,0 +1,97 @@
+Service overview
+================
+
+This article gives you an overview of the service plans and features available in Aiven for Apache Flink beta.
+
+You can run Aiven for Apache Flink on AWS or GCP cloud environments. Various deployment options are available, ranging from fully managed access over the public internet to subnet deployments under your own cloud account. All services and infrastructure are single-tenant and dedicated solely to a single customer.
+
+Aiven offers single-node and three-node Apache Flink clusters:
+
+* Single-node clusters are not highly available, but support automatic failover. They start from 4GB RAM/2 vCPU and extend up to 32GB/8 vCPU. This setup is a good option for proof of concepts and preliminary testing for development purposes.
+
+* Three-node clusters are highly available, support automatic failover, and range from 4GB RAM/2 vCPU to 32GB/8 vCPU.
+
+Our product page provides you detailed information about the available plans and pricing. 
```

Comment:
Is there a product page out there already? (I guess not)

Reply:
At least nothing published yet, but I'm planning on adding a link to that once it is published ;)